packages/mds-stream-processor/connectors/kafka-connector.ts

```diff
@@ -15,16 +15,20 @@
  */
 
 import logger from '@mds-core/mds-logger'
-import stream from '@mds-core/mds-stream'
-import { StreamConsumerOptions, StreamProducerOptions } from '@mds-core/mds-stream/kafka/helpers'
+import {
```

Comment:
How portable are imports like this? I've noticed some issues recently when doing a similar style import from an external repo.

Reply:
I left them that way to find out. I suspect they won't work, but they are also exported from the mds-stream/index.ts.